.tidyrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "indent": 2,
+  "ribbon": 1,
+  "typeArrowPlacement": "first",
+  "unicode": "never",
+  "width": 2147483647
```

Comment:
This one seems like it should maybe be omitted, as this is a weird default value to produce.

Reply:
I would prefer to not omit any options when generating an RC file. That way it will always work in the future even if we change a default. We can change the default to a `Maybe` value or something though.

Reply:
I feel like that would be an improvement.